cmd/svcat/testdata/responses/clusterservicebrokers.json

```diff
@@ -30,8 +30,8 @@
             "type": "Ready",
             "status": "True",
             "lastTransitionTime": "2018-01-11T20:53:31Z",
-            "reason": "FetchedCatalog",
-            "message": "Successfully fetched catalog entries from broker."
+            "reason": "FetchedClusterCatalog",
```

Comment:
We shouldn't change this - it will break existing users of the API and I doubt this distinction needs to surface to users in any event.